### ISSUE TITLE
NotBeInRange

### ISSUE THREAD

**Issue by jnyrup** [fluentassertions/fluentassertions#595]
There does not seem to be support for the negation of `BeInRange(T minimumValue, T maximumValue)`.

I propose addition of `NotBeInRange(T minimumValue, T maximumValue)`.

If the proposal is welcomed, I'll be happy to submit a PR once 5.0 is merged into master (unless somebody beats me to it).

**Comment by dennisdoomen** [comment]
As with #594, go ahead.
